src/imagery/i.eodag/i.eodag.py

```diff
@@ -55,7 +86,7 @@
 # % key: id
 # % type: string
 # % multiple: yes
-# % description: List of scenes IDs to download
+# % description: List of scenes IDs to download | A single text file with one ID on each line
```

Comment:
I think id and text file should be two different options. `id` take multiple values, `file` just one. 
`id` or `file` should be present if `l` is not used (this means download data)

Reply:
I think it should also be possible to not set any of them, when the `l` flag is not used... that should mean search for datasets, and download them right away, or is that not practical?

Reply:
@HamedElgizery yes it works

Reply:
Should be working now. I also just added the option to be able to list the products that was found by searching with IDs, instead of just always proceeding to download.